modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -19,6 +19,7 @@ class NoticeOfDisagreement < ApplicationRecord
     attr_readonly :form_data
 
     before_create :assign_metadata
+    before_update :submit_evidence_to_central_mail, if: -> { status_changed_to_success? && delay_evidence_enabled? }
```

Comment:
![stick-around-bob-ross-gif-by-originals](https://github.com/department-of-veterans-affairs/vets-api/assets/8226466/cec11e15-c67f-46f3-b3fc-4dc81380aa33)